website1/css/main.css

```diff
@@ -0,0 +1,191 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    font: inherit;
+    vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+    display: block;
+}
+
+body {
+    line-height: 1;
+}
+
+ol, ul {
+    list-style: none;
+}
+
+blockquote, q {
+    quotes: none;
+}
+
+blockquote:before, blockquote:after,
+q:before, q:after {
+    content: '';
+    content: none;
+}
+
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+
+.header {
+    position: absolute;
+    width: 1440px;
+    height: 81px;
+    left: 0;
+    top: 0;
+    align-items: center;
+
+    background: rgba(255, 255, 255, 0.6);
+    backdrop-filter: blur(25px);
+}
+
+.logo_company {
+    width: 151px;
+    height: 81px;
+    margin-left: 75px;
+}
+
+.name_company {
+    position: absolute;
+    width: 286px;
+    height: 29px;
+    left: 226px;
+    top: 26px;
+
+    /*font-family: 'NEXT ART';*/
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 159.02%;
+    color: #4F4A45;
+
+}
+
+.menu a {
+    color: #4F4A45;
+    width: auto;
+    text-decoration: none;
+
+}
+
+.menu li {
+    margin-right: 20px;
+}
```

Comment:
Tag referencing in BEM?
`li` or `a` tags often are elements, that need to add a class for him.